src/App.tsx

```diff
@@ -12,161 +12,34 @@ export const App = () => (
     >
       <div className="container">
         <div className="navbar-brand">
-          <a className="navbar-item" href="#/">
+          {/* need to replace a to Link */}
+          <NavLink
+            to="/"
+            className={
+              ({ isActive }) => classNames('navbar-item', {
+                'has-background-grey-lighter': isActive,
+              })
+            }
+          >
             Home
-          </a>
-
-          <a
-            className="navbar-item has-background-grey-lighter"
-            href="#/people"
+          </NavLink>
+          <NavLink
+            to="/people"
+            className={
+              ({ isActive }) => classNames('navbar-item', {
+                'has-background-grey-lighter': isActive,
+              })
```

Comment:
this code is duplicated, so make sense to move it to the separate function and reuse it
`({ isActive }) => classNames('navbar-item', {
                'has-background-grey-lighter': isActive,
              })`